<<<<<<< HEAD
<?xml version="1.0"?>
<robot xmlns:xacro="http://ros.org/wiki/xacro">
  <property name="M_PI" value="3.14159274"/>

  <!-- common color -->
  <xacro:macro name="set_color">
	  <material name="green">
   		<color rgba="0.0 1.0 0.0 1.0"/>
  	  </material>
  </xacro:macro>

  <!-- calculate box inertia -->
  <xacro:macro name="box_inertial" params="m d w h">
    <mass value="${m}"/>
    <inertia ixx="${(m * (h * h + d * d)) / 12}"   ixy="0" ixz="0"
             iyy="${(m * (w * w + d * d)) / 12}"   iyz="0"
             izz="${(m * (w * w + h * h)) / 12}" />
  </xacro:macro>

</robot>
=======
<?xml version="1.0"?>
<robot xmlns:xacro="http://ros.org/wiki/xacro">
  <property name="M_PI" value="3.14159274"/>

  <!-- common color -->
  <xacro:macro name="set_color">
	  <material name="green">
   		<color rgba="0.0 1.0 0.0 1.0"/>
  	  </material>
  </xacro:macro>

  <!-- calculate box inertia -->
  <xacro:macro name="box_inertial" params="m d w h">
    <mass value="${m}"/>
    <inertia ixx="${(m * (h * h + d * d)) / 12}"   ixy="0" ixz="0"
             iyy="${(m * (w * w + d * d)) / 12}"   iyz="0"
             izz="${(m * (w * w + h * h)) / 12}" />
  </xacro:macro>

</robot>
>>>>>>> 42ca683593df3d34624e90336fab16e0fade033c
